tasks/MeasureTasks.py

```diff
@@ -0,0 +1,51 @@
+import astropy.units as u
+import numpy as np
+import lsst.pipe.base as pipeBase
+import lsst.pex.config as pexConfig
+from lsst.verify import Measurement
+import lsst.pex.config as pexConfig
+from sst_metrics_utils.filtermatches import filterMatches
+from lsst.validate.drp.repeatability import calcPhotRepeat
+
+class NumSourcesTask(pipeBase.Task):
+
+    ConfigClass = pexConfig.Config
+    _DefaultName = "numSourcesTask"
+
+    def run(self, matchedCatalog):
+        self.log.info(f"Counting sources in matched catalog")
+        nSources = len(matchedCatalog)
+        meas = Measurement("nsrcMeas", nSources * u.count)
+        return pipeBase.Struct(measurement=meas)
+
+
+class PA1TaskConfig(pexConfig.Config):
+    brightSnrMin = pexConfig.Field(doc="Minimum median SNR for a source to be considered bright.",
```

Comment:
Can we set this value in the yaml config file? It might be nice to show as an example.

Reply:
Yes.  We can put an example in.